config/bert.yaml

```diff
@@ -2,38 +2,39 @@ run:
     ###################################################
     # Generic configurations options related to
     #  handling of experiments
-
-    experiment_name: BERT-Estimator WMT20 Target en_de.nmt
+    experiment_name: BERT WMT20 EN-DE
     seed: 42
-    mlflow_tracking_uri: mlruns/
-    # output_dir: null  # mlflow will create one if this is null
     use_mlflow: false
 
 trainer:
     ###################################################
     # Generic options related to the training process
     #  that apply to all models
-
+    deterministic: true
     gpus: -1
-    log_interval: 100
     epochs: 10
 
     main_metric:
+        - WMT19_MCC
         - PEARSON
-        - WMT19_F1_MULT
 
+    gradient_max_norm: 1.
+    gradient_accumulation_steps: 1
+
```

Comment:
```suggestion
    # Setting amp_level to O1 to use mixed-precision training
```

Reply:
It's the only option that is not self-explanatory

Reply:
Alright, added info and a PTL reference